tests/sawtooth-pbft-test.dockerfile

```diff
@@ -35,10 +35,10 @@ RUN echo "deb [arch=amd64] http://repo.sawtooth.me/ubuntu/nightly bionic univers
     python3-sawtooth-cli \
     python3-sawtooth-intkey \
     python3-sawtooth-rest-api \
-    python3-sawtooth-settings \
```

Comment:
Is this change related?

Reply:
Yes. 1-3 images do not include the python version of sawtooth-settings so it need to be replaced with the rust version which is included. I updated the commit to clarify these.